data/json/items/comestibles/med.json

```diff
@@ -87,7 +87,7 @@
         "type" : "COMESTIBLE",
         "comestible_type" : "MED",
         "name" : "antibiotic",
-        "description" : "Antibacterial medication designed to prevent or stop the spread of infection.",
+        "description" : "A strong antibacterial medication designed to prevent or stop the spread of infection.",
```

Comment:
Good catch updating the existing antibiotic description as well :+1:
Maybe actually call out that it immediately cures infections?